### DISCUSSION TITLE
How to merge asset downstream of factory-created groups

### DISCUSSION THREAD

**Discussion by dagsir** [dagster-io/dagster#17508] [Q&A] [answered]
Hi everyone! How should I approach this use case of ours? We are using a factory design approach to our pipelines. How can I “merge” (sorry for the lack of a better term) them to run an independent asset outside the different asset groups?

Here’s an image to better clarify what we have. We have these three groups, and I want to run a specific asset once they finish materializing a particular asset in their group. How can I achieve that? Thank you!

![image](https://github.com/dagster-io/dagster/assets/654855/66e3e351-ecbf-4744-8c04-4c9ce431d528)

---

*The question was originally asked in [Dagster Slack](https://dagster.slack.com/archives/C01U954MEER/p1698717394053619?thread_ts=1698717394.053619&cid=C01U954MEER).*

**Comment by yuhan** [comment, marked as answer]
You can define a [multi-asset sensor](https://docs.dagster.io/concepts/partitions-schedules-sensors/asset-sensors#multi-asset-sensors) which can trigger downstream based on multiple asset materialization events.
